packages/block-editor/src/store/actions.js

```diff
@@ -201,16 +220,41 @@ export function toggleSelection( isSelectionEnabled = true ) {
  *
  * @param {(string|string[])} clientIds Block client ID(s) to replace.
  * @param {(Object|Object[])} blocks    Replacement block(s).
- *
- * @return {Object} Action object.
```

Comment:
`@yields` is available as a valid substitute in JSDoc.

http://usejsdoc.org/tags-yields.html